fetch-utils.js

```diff
@@ -5,14 +5,15 @@ const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export async function getDogs() {
     // from the dogs table, select all items
-
+    const response = await client.from('dogs').select('*');
+    //console.log(response);
     // and return the response
     return response.data;    
 }
 
 export async function getDog(id) {
     // from the dogs table, select a single dog who has the matching id
-
+    const response = await client.from('dogs').select('*').match({ id }).single(); //is the .match(id) as in it needs ({ id })wrong?
```

Comment:
Julie had written out the match like .match({ id: id }) - but it looks like this is working because your variable name(id) and the key we're matching (id) are named the same!